src/ansys/aedt/core/visualization/post/common.py

```diff
@@ -0,0 +1,2417 @@
+            Report Context.
+            The default is ``None``, in which case the default context is used.
+            For Maxwell 2D/3D Eddy Current solution types this can be provided as a dictionary
+            where the key is the matrix name and value the reduced matrix.
+        is_siwave_dc : bool, optional
+            Whether if the setup is Siwave DCIR or not. Default is ``False``.
+
+        Returns
+        -------
+        dict
+            A dictionary with primary key the report type, secondary key the solution type and
+            third key the report categories.
+        """
+        rep_quantities = {}
+        for rep in self.available_report_types:
+            rep_quantities[rep] = {}
+            solutions = [solution] if isinstance(solution, str) else self.available_report_solutions(rep)
+            for solution in solutions:
+                rep_quantities[rep][solution] = {}
+                for quant in self.available_quantities_categories(
+                    rep, context=context, solution=solution, is_siwave_dc=is_siwave_dc
+                ):
+                    rep_quantities[rep][solution][quant] = self.available_report_quantities(
+                        rep, quantities_category=quant, context=context, solution=solution, is_siwave_dc=is_siwave_dc
+                    )
+
+        return rep_quantities
+
+    @pyaedt_function_handler()
+    def available_report_solutions(self, report_category=None):
+        """Get the list of available solutions that can be used for the reports.
+        This list differs from the one obtained with ``app.existing_analysis_sweeps``,
+        because it includes additional elements like "AdaptivePass".
+
+        Parameters
+        ----------
+        report_category : str, optional
+            Report Category. Default is ``None`` which takes default category.
+
+        Returns
+        -------
+        list
+
+        References
+        ----------
+        >>> oModule.GetAvailableSolutions
+        """
+        if not report_category:
+            report_category = self.available_report_types[0]
+        if report_category:
+            return list(self.oreportsetup.GetAvailableSolutions(report_category))
+        return None  # pragma: no cover
+
+    @pyaedt_function_handler()
+    def _get_plot_inputs(self):
+        names = self._app.get_oo_name(self.oreportsetup)
+        plots = []
+        skip_plot = False
+        if self._app.design_type == "Circuit Netlist" and self._app.desktop_class.non_graphical:
+            skip_plot = True
+        if names and not skip_plot:
+            for name in names:
+                obj = self._app.get_oo_object(self.oreportsetup, name)
+                report_type = obj.GetPropValue("Report Type")
+
+                if report_type in TEMPLATES_BY_NAME:
+                    report = TEMPLATES_BY_NAME[report_type]
+                else:
+                    report = ansys.aedt.core.visualization.report.standard.Standard
```

Comment:
```suggestion
                report = TEMPLATES_BY_NAME.get(report_type, TEMPLATES_BY_NAME["Standard"])
```
Minor change for maintenance.